src/inc/corinfo.h

```diff
@@ -2386,6 +2387,7 @@ class ICorStaticInfo
 
     virtual void getReadyToRunHelper(
             CORINFO_RESOLVED_TOKEN * pResolvedToken,
+            CORINFO_LOOKUP_KIND *    pLookupKind,
```

Comment:
These should be just one out argument that describes the lookup. The existing `CORINFO_CONST_LOOKUP` argument should be changed to contain what you need - changing the type of the argument to `CORINFO_LOOKUP` may be best.